package.json

```diff
@@ -11,10 +11,16 @@
     "url": "git+https://github.com/ngParty/Angular1-scaffold.git"
   },
   "scripts": {
-    "typings": "typings install",
-    "postinstall": "npm run typings",
-    "start": "webpack-dev-server --port $npm_package_config_port --hot --inline --progress --profile --colors --watch --display-error-details --display-cached --content-base src/",
-    "build": "NODE_ENV=production webpack --config webpack-build.config.js"
+    "rimraf": "rimraf",
+    "typings": "typings",
+    "postinstall": "npm run typings -- install",
+    "start": "npm run server",
+    "build": "NODE_ENV=production webpack --config webpack-build.config.js",
+    "build:dev": "webpack --progress --profile --colors --display-error-details --display-cached",
```

Comment:
well I think the issue culprit is your original config ( path to root ), we should use relative paths for + combined with `__dirname` and `path.resolve` == cross platform.

I'll send updated config with next commit batch

```
 output: {
    /**
     * The output directory as absolute path (required).
     *
     * See: http://webpack.github.io/docs/configuration.html#output-path
     */
    path: '/',

    publicPath: '/',

    /**
     * Specifies the name of each output file on disk.
     * IMPORTANT: You must not specify an absolute path here!
     *
     * See: http://webpack.github.io/docs/configuration.html#output-filename
     */
    filename: '[name].js',

    /**
     * The filename of the SourceMaps for the JavaScript files.
     * They are inside the output.path directory.
     *
     * See: http://webpack.github.io/docs/configuration.html#output-sourcemapfilename
     */
    sourceMapFilename: '[name].map'
  }
```